fe/fe-core/src/main/java/com/starrocks/sql/ast/ManualRefreshSchemeDesc.java

```diff
@@ -0,0 +1,14 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.sql.ast;
+
+import com.starrocks.catalog.RefreshType;
+
+public class ManualRefreshSchemeDesc extends RefreshSchemeDesc {
```

Comment:
Why ManualRefreshSchemeDesc and SyncRefreshSchemeDesc need to be single classes， which seems are strange.

Reply:
We have AsyncRefreshSchemeDesc、ManualRefreshSchemeDesc and SyncRefreshSchemeDesc which are extended RefreshSchemeDesc. AsyncRefreshSchemeDesc have startTime and intervalLiteral property. I think ManualRefreshSchemeDesc or SyncRefreshSchemeDesc may have properties like AsyncRefreshSchemeDesc in the future.

Reply:
Ok. I think if we have Refresh type in RefreshSchemeDesc, we needn't ManualRefreshSchemeDesc and SyncRefreshSchemeDesc.  which is a little issue, both ways are OK.